docs/pql_examples.rst

```diff
@@ -246,6 +246,19 @@ Supported transform functions
    format ``1:SECONDS:EPOCH`` with a granularity of ``15:MINUTES`` (*i.e.* nearest 15-minute value lower than the value
    of ``date`` column.
 
+``DATETRUNC``
+   (Presto) SQL compatible date truncation, equivalent to the Presto function `date_trunc
```

Comment:
Presto, PostgreSQL and Redshift all supports DATE_TRUNC, and all of them take two arguments: DATE_TRUNC('unit', timestamp). What is the specific reason why we need 4 or 5 arguments? Should we also add support for 2 arguments, where timestamp is millisSinceEpoch?